app/components/home/balance-card.tsx

```diff
@@ -55,18 +63,26 @@ export const BalanceCard: FC<BalanceCardProps> = props => {
           <ArrowIcon direction="down" mr="base-tight" />
           Receive
         </Button>
-        <Button mode="secondary" size="md" ml="tight" onClick={requestTestnetStacks}>
-          <Box
-            mr="extra-tight"
-            fontSize="18px"
-            left="-4px"
-            position="relative"
-            display={['none', 'none', 'block']}
+        {NETWORK === 'testnet' && (
+          <Button
+            mode="secondary"
+            size="md"
+            ml="tight"
+            isDisabled={requestingTestnetStx}
+            onClick={e => requestTestnetStacks(e)}
           >
-            🚰
-          </Box>
-          {requestingTestnetStx ? 'Requesting faucet' : 'Get testnet STX'}
-        </Button>
+            <Box
```

Comment:
Maybe add a tooltip that provides guidance for alt-clicking?

Reply:
But it's secret

Reply:
I'm not sure it _has_ to be a secret. Community members may want to test Stacking as well, and I assume we can support several of them per day hitting this button?

Reply:
Alright, will add a toolstip

Reply:
See d585e6e